src/main/resources/META-INF/plugin.xml

```diff
@@ -1,23 +1,36 @@
 <idea-plugin require-restart="true">
     <id>fi.aalto.cs.intellij-plugin</id>
     <name>A+ Courses</name>
-    <vendor email="intellij-plugin-letech@aalto.fi" url="https://research.cs.aalto.fi/LeTech/">LeTech Group at Aalto University</vendor>
+    <vendor email="intellij-plugin-letech@aalto.fi"
+            url="https://research.cs.aalto.fi/LeTech/">LeTech Group at Aalto University</vendor>
+    <!--  this is the range of supported IntelliJ IDEA version the plugin can be used with  -->
+    <idea-version since-build="202.6397.94"/>
```

Comment:
Does this mean versions earlier than 2020.2 won't work?

Reply:
yes, my thinking is, that we explicitly aim the latest available major of IDEA. as the students are to use CE, then anyone can get 2020.2 wo any issues (or upgrade seamlessly to it)